packages/flutter_driver/test/src/wait_test.dart

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/src/common/wait.dart';
+
+import '../common.dart';
```

Comment:
Why import one relative and one absolute?

Reply:
Was following other test files... This common.dart also lives under the test folder, but one level up. Shall I always use absolute path if the import is not in the same folder?

Reply:
Note this is a test file, and using `import '../../lib/src/common/wait.dart';` makes the analyzer complain "Avoid relative imports for files under lib/".